packages/analytics-v1.1/src/core/analytics.js

```diff
@@ -293,6 +284,18 @@ class Analytics {
         }
       }
 
+      // set loaded flag to true
+      this.loaded = true;
+
+      // Execute onLoaded callback if provided in load options
+      if (this.options && typeof this.options.onLoaded === 'function') {
+        this.options.onLoaded(this);
+      }
+
+      // Execute any pending buffered requests
+      // (needed if the load call was not previously buffered)
+      processDataInAnalyticsArray(this);
+
```

Comment:
Let's move this block to the end of the function. It's the ideal place as the SDK is now loaded and en route to being ready.

Also, please verify that no other blocks of code before it relies on the `loaded` flag to be `true`.

Reply:
Which block? line 287-297?

Reply:
Yes.

Reply:
Are we good with this one now?